bundles/org.openhab.binding.folderwatcher/src/main/resources/OH-INF/i18n/folderwatcher.properties

```diff
@@ -45,6 +47,20 @@ thing-type.config.folderwatcher.localfolder.localDir.label = Local Directory
 thing-type.config.folderwatcher.localfolder.localDir.description = Local directory to be watched
 thing-type.config.folderwatcher.localfolder.pollIntervalLocal.label = Polling Interval
 thing-type.config.folderwatcher.localfolder.pollIntervalLocal.description = Interval for polling folder changes, sec
+thing-type.config.folderwatcher.s3bucket.S3BucketName.label = S3 bucket name
```

Comment:
Probably needless to say, but the i18n file needs to be updated if you change the config to camelcasing